content/en/logs/explorer/analytics/_index.md

```diff
@@ -45,7 +45,7 @@ When aggregating indexed logs by **Fields**, all logs matching your query filter
 On top of these aggregates, you can extract the following measures:
 
 - **count of logs** per group
-- **unique count** of coded values for a query search value per group
+- **count unique of** coded values for a query search value per group
```

Comment:
```suggestion
- **count of unique coded values** for a query search value per group
```

I'm actually not sure if this would be the correct verbiage, but "count unique of coded values" just doesn't sound right to me

Reply:
@cswatt it didn't sound right to me either..i just really wanted it to work because that is what's in the UI and users don't seem to realize this is related to that element. What if I just added a note:

> **count of unique coded values** for a query search value per group (shown in the UI as `count unique of`